pom.xml

```diff
@@ -1428,6 +1426,10 @@
             <groupId>org.codehaus.groovy</groupId>
             <artifactId>groovy-all</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
Both property values are valid only for scala 2.10.x, 2.11.x does no longer provide or use org.scala-lang:jline. If you want  to keep it, the values should reflect the default scala version which uses jline:jline:2.12.1.